src/auth/auth.go

```diff
@@ -112,6 +112,9 @@ const (
 
 	// ProjectCreatorRole is a role which grants the ability to create projects
 	ProjectCreatorRole = "projectCreator"
+
+	// JobAdminRole is a role which grants access to PJS RPCs without needing job contexts.
+	JobAdminRole = "jobAdmin"
```

Comment:
I'm a little worried that the auth system thinks Jobs are JobSet jobs and not PJS jobs.  What did the design doc say about using the same name for two different things?

Reply:
I understand the concern. I think though we might be able to handle that via prefixes.